test/config/utility.cc

```diff
@@ -114,6 +113,18 @@ void ConfigHelper::finalize(const std::vector<uint32_t>& ports) {
     }
   }
   ASSERT(port_idx == ports.size());
+
+  if (!connect_timeout_set_) {
```

Comment:
Alternately you could just set it in initialize() before anyone has had a chance to override.

Reply:
I put it at the end of finalize() so that it sees any additional clusters that the test might create via addConfigModifier.